GameOfHive/HiveButton.swift

```diff
@@ -59,6 +59,11 @@ class HiveButton: UIButton {
         backgroundView.backgroundColor = UIColor.menuButtonBackgroundColor
         backgroundView.userInteractionEnabled = false
     }
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        backgroundView.frame = bounds
+    }
```

Comment:
op mijn 5S gebeurt er anders dit:
![screenshot journal 2016-04-05 at 12 08 07](https://cloud.githubusercontent.com/assets/1320087/14279592/601853c8-fb2e-11e5-86c2-ff8954039447.PNG)

de hexagon in de background vult niet de hele button